endpoints/endpoint.go

```diff
@@ -0,0 +1,79 @@
+package endpoint
+
+import (
+	"encoding/json"
+	"math/rand"
+	"net/http"
+	"strings"
+	"time"
+	"fmt"
+)
+
+const shortUrlLength = 4
+
+type StoreHandler struct {
+	Store map[string]string
+}
+
+func NewstoreHandlers() *StoreHandler {
+	return &StoreHandler{
+
+		Store: map[string]string{},
+	}
+
+}
+func (u *StoreHandler) ShortenUrl(w http.ResponseWriter, r *http.Request) {
+	
+	CheckUrl := strings.Split(r.URL.Path, "/")[2]
```

Comment:
Since this variable is not something that is exported, it should be 
```suggestion
	checkUrl := strings.Split(r.URL.Path, "/")[2]
```